go/client/cmd_team_list_memberships.go

```diff
@@ -193,7 +193,14 @@ func (c *CmdTeamListMemberships) runUser(cli keybase1.TeamsClient) error {
 			role += strings.ToLower(t.Role.String())
 		}
 		if c.showAll {
-			fmt.Fprintf(c.tabw, "%s\t%s\t%s\t%s\n", t.FqName, role, t.Username, t.FullName)
+			var reset string
+			if !t.Active {
+				reset = "(inactive due to account reset)"
```

Comment:
We want to show this information?  Does it matter?

Reply:
We get it when we call UIDMapper to get full name, so might as well show it. Fine for me either way. @malgorithms  ?